api/src/main/java/au/org/aodn/nrmn/restapi/dto/species/SpeciesDto.java

```diff
@@ -21,4 +23,10 @@
 
     private String order;
 
+    private String rank;
+
+    private Boolean superceded;
+
+    private String validName;
```

Comment:
Is `validName` the name of the species which has superseded this one?

Reply:
Yes

Reply:
I can include the aphiaId for that as well.  I'll just add that now

Reply:
Actually, just managed to get a hold of @bpasquer  and she only wants superseded by added as a column after species name.  Removing displayName and rank.

Reply:
Otherwise existing functionality is OK